addons/mail/models/res_role.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class ResRole(models.Model):
+    _name = "res.role"
+    _description = "Represents a role in the system, which can be used to categorize users." \
+    "Each role has a unique name and can be associated with multiple users." \
+    "Roles can be pinged in the message, and users associated with the role will receive a notification."
+
+    name = fields.Char(required=True, translate=True)
+    user_ids = fields.Many2many("res.users", string="Users")
```

Comment:
Shouldn't we define the relation table name so that it matches on both sides?

Reply:
is it needed? from what I see, it is correctly computed in the database

Reply:
I think it's computed correctly if there is only one many relation towards each model respectively, but it gets lost if we add more.

It's a bit too magic for me, when we can just specify `rel="res_role_res_users_rel"` and be sure of what will happen (and we can easily grep it in code to find the inverse).